modules/microshift-data-cleaning-full-cleanup.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * microshift_troubleshooting/microshift-cleanup-data.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-data-cleaning-full-cleanup_{context}"]
+= Cleaning all data and configuration
+
+Cleanup all the {microshift-short} data and configuration by running the following script. This script completes the following cleanup actions:
+
+* Stop and disable {microshift-short} services
+* Delete all {microshift-short} pods
+* Delete all container image storage
+* Reset network configuration
+* Delete the /var/lib/microshift data directory
+* Delete OVN configuration
+
+.Procedure
+
+. Execute the `microshift-cleanup-data` script, by running the following command.
++
+[WARNING]
+====
+This option deletes all {microshift-short} and user-specific data. Use with caution.
+====
++
+[source,terminal]
+----
+$ sudo microshift-cleanup-data --all
+----
++
+[NOTE]
+====
+The script prompts you with a message to confirm the operation. Type 1 or Yes to continue, or anything else to abort the cleanup.
+====
++
+.Example output when you abort the cleanup
```

Comment:
I think these examples should be reversed because we tell users how to continue in the Note first.